### DISCUSSION TITLE
30. January 2024

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1730] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [ ] Dmitrii: design overview of AEX Notify https://github.com/gramineproject/gramine/pull/1530
- [ ] Dmitrii: pinging Chia-Che to approve https://github.com/gramineproject/examples/pull/90

**Comment by woju** [comment]
- [x] @woju: we may need 1.6.1 release before Ubuntu 24.04 LTS because of cryptography.io: #1731 / #1732 (if we won't manage to release 1.7 before April)

**Reply by dimakuv** [reply]
UPDATE: The bug was fixed by cryptography.io in v4.0.2, and this version (or higher) will be incorporated in OS distros like Ubuntu 24.04. Nothing to be done from Gramine side.

**Comment by dimakuv** [comment]
Random notes:

### Paper on AEX Notify

https://www.usenix.org/system/files/usenixsecurity23-constable.pdf

### Note from Scott Raynor

"HPE ProLiant Gen11 servers featuring 5th Gen Intel® Xeon® Scalable processors will be available to order, worldwide, on February 4, 2024, through HPE and HPE’s channel partners. They will also be available through the HPE GreenLake cloud platform."

### One more company that uses Gramine

https://writings.flashbots.net/suave-tee-coprocessor
